src/vehicle.cpp

```diff
@@ -1057,10 +1071,19 @@ void vehicle::use_controls()
         menu.addentry( toggle_camera, true, 'M', camera_on ?
                        _("Turn off camera system") : _("Turn on camera system") );
     }
+    if( has_plow ){
+        menu.addentry( toggle_plow, true, MENU_AUTOASSIGN, _("Toggle Plow"));
+    }
+    if( has_planter ){
+        menu.addentry( toggle_planter, true, 'P', _("Toggle Planter"));
```

Comment:
There is a bit of inconsistency with the vehicle controls menu: some options say "toggle X", others say "enable X" or "disabled X", depending on the current state. The second version should probably be used for all options as it gives the player a way to check the current status. Currently there is now way for the player to check whether the plow or planer is on.